src/components/Article/Article.jsx

```diff
@@ -9,7 +9,11 @@ import moment from 'moment'
 export const Article = ({ article, setArticle, user }) => {
     let { article_id } = useParams()
     const [isLoading, setIsLoading] = useState(true)
+    const [leaveComment, setLeaveComment] = useState(false)
+    const [posting, setPosting] = useState(false)
+    let [postIndicator, setPostIndicator] = useState(0)
```

Comment:
Is there a specific reason why let is used here in destructuring from useState? const declarations should be first choice when we aren't reassigning values purposefully because the set state function (setPostIndicator) is what we're using to set the state value (postIndicator) so use of let shouldn't be necessary/could open up room for error in the rest of our code as the set state function relies on closure